src/schemas.jl

```diff
@@ -165,6 +189,25 @@ written via [`Legolas.write`](@ref).
 """
 identifier(sv::SchemaVersion) = throw(UnknownSchemaVersionError(sv))
 
+"""
+    Legolas.schema_provider(::SchemaVersion)
+
+Returns a NamedTuple with keys `name` and `version`. The name is a `Symbol` corresponding to the package which defines the schema version, if known; otherwise `nothing`. Likewise the `version` is a `VersionNumber` or `nothing`.
+"""
+schema_provider(::SchemaVersion) = (; name=nothing, version=nothing)
```

Comment:
Why do we switch to `nothing` here?

Reply:
I switched to missing elsewhere since that’s what `lift` returns and I wanted to use `lift`